packages/core/src/components/tag/tag.tsx

```diff
@@ -101,6 +108,7 @@ export class Tag extends React.PureComponent<ITagProps, {}> {
             Classes.intentClass(intent),
             {
                 [Classes.ACTIVE]: active,
+                [Classes.FILL]: fill,
```

Comment:
there is no corresponding CSS change, so this doesn't seem to do anything

Reply:
https://github.com/palantir/blueprint/blob/develop/packages/core/src/common/_flex.scss#L8

Reply:
are you able to get it working in the docs? I can't

Reply:
The parent in the docs isn't a `@pt-flex-container`, so no.

Why does `.pt-flex` only work inside a `@pt-flex-container`? Why not just globally allow it? I doubt consumers use the mixin, I certainly never have.

Reply:
i added this prop to support flex containers, as you've noticed. i'm happy to add some corresponding styles (`display: flex; width: 100%`) if you'd like to see that too @adidahiya @invliD?

Reply:
yeah, I would like to see some way to enable this style modifier in the docs, thanks

Reply:
done